### PR TITLE
custom user model created

### PR DESCRIPTION
**Pull request by anarbyrm**

### CODE REVIEW

**Comment by anarbyrm** [comment]
conflict resolved

**Comment by anarbyrm** [comment]
All codes you requested have been changed.

File: cargo_backend/accounts/models.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    def create_superuser(self, email, password, **extra_fields):
+        extra_fields.setdefault('is_staff', True)
+        extra_fields.setdefault('is_superuser', True)
+        extra_fields.setdefault('is_active', True)
+        extra_fields.setdefault('fin_code', '0000000')
+        extra_fields.setdefault('gov_id_prefix', 'AZE')
+        extra_fields.setdefault('gov_id', '00000000')
+        extra_fields.setdefault('gender', 'male')
+        extra_fields.setdefault('phone', '1234567')
+        extra_fields.setdefault('birth_date', timezone.now)
```
Comment (mmzeynalli) [inline cargo_backend/accounts/models.py:30]: should not it be `timezone.now()`?

File: cargo_backend/accounts/tests/user_factory.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+class UserModelFactory(DjangoModelFactory):
+    class Meta:
+        model = User
+
+    email = generate_email()
+    password = "testing_123"
+    gender = 'male'
+    phone = generate_phone_and_fin()
```
Comment (mmzeynalli) [inline cargo_backend/accounts/tests/user_factory.py:32]: oh, you do not need this. Just add fields that are almost always required. If you remove phone, gender from factory, you can still create object like `UserModelFactory(email=x, gender='male', phone='xxx')`

**Review by mmzeynalli — CHANGES_REQUESTED** [review]

**Review by mmzeynalli — APPROVED** [review]

**Pull request merged**